npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -316,7 +316,7 @@
         }]
       end
       let(:previous_version) { "d5ac0584ee9ae7bd9288220a39780f155b9ad4c8" }
-      let(:version) { "0c6b15a88bc10cd47f67a09506399dfc9ddc075d" }
+      let(:version) { "98e8ff1da1a89f93d1397a24d7413ed15421c139" }
```

Comment:
Oh why did this change?

Reply:
Ah this is the latest commit? hm not sure we should change this variable for all tests as we then loose code paths testing updating to specific commits vs latest when there's no reference. I think we still want some tests that try to update to a specific old commit, e.g. pointing to a tag.

Reply:
> Oh why did this change?

The fixtures in those tests do not pin to a hash (e.g. `"is-number": "jonschlinkert/is-number"`) so the npm install step fetches the latest hash for the default branch of that repo.

> Ah this is the latest commit?

Yep. 

> hm not sure we should change this variable for all tests as we then loose code paths testing updating to specific commits vs latest when there's no reference

🤔 Can I update the constraint in the package.json for those fixtures to pin to a fixed version?